nutkit/protocol/requests.py

```diff
@@ -214,8 +212,19 @@ class BookmarksConsumerCompleted:
     Signal the method call has finished
     """
 
-    def __init__(self, requestId):
+    def __init__(self, requestId, bookmarkManagerId):
```

Comment:
No need to send the bmm id. The backend just initiated this callback. It shouldn't be too hard for it to remember which BMM it was that triggered the callback. So I vote for having only a single source of truth (i.e., `requestId`). (Same comment for `BookmarksSupplierCompleted`).